cmd/queue/main.go

```diff
@@ -169,6 +169,11 @@ func proxyHandler(breaker *queue.Breaker, stats *network.RequestStats, tracingEn
 }
 
 func preferPodForScaledown(downwardAPILabelsPath string) (bool, error) {
+	// If downwardAPILabelsPath is empty, feature is disabled.
+	if downwardAPILabelsPath == "" {
+		return false, nil
+	}
+
 	// Short circuit a rejection when no label path file is mounted
 	if _, err := os.Stat(downwardAPILabelsPath); os.IsNotExist(err) {
```

Comment:
I believe we can straight out drop this one then? I think we have an either/or situation now :thinking:

Reply:
yep, think you're right, there's no case where this fails now, and this way we even get to skip the stat when the feature _is_ enabled \o/